src/internal/SelectionLabel.js

```diff
@@ -16,13 +16,20 @@ export const styleSheet = createStyleSheet('SelectionLabel', () => {
     reverse: {
       flexDirection: 'row-reverse',
     },
+    disabled: {
+      opacity: 0.5,
```

Comment:
Might be good to have a theme variable for this disabled opacity value.
Notice that it's `0.4` on the `Button`. But I'm wondering where we should put it.